doc/reference/api/TSNetVConnSSLCachedHitSet.en.rst

```diff
@@ -0,0 +1,32 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed
+   with this work for additional information regarding copyright
+   ownership.  The ASF licenses this file to you under the Apache
+   License, Version 2.0 (the "License"); you may not use this file
+   except in compliance with the License.  You may obtain a copy of
+   the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+   implied.  See the License for the specific language governing
+   permissions and limitations under the License.
+
+
+TSNetVConnSSLCachedHitSet
+============
+
+Synopsis
+--------
+
+`#include <ts/ts.h>`
+
+.. c:function:: void TSNetVConnSSLCachedHitSet(TSVConn vc, int state)
+
+Description
+-----------
+
+   Mark an SSL Connection as reused, which means that an SSL Session Cache or an SSL Ticket have been used to prevent a full handshake for this SSL Connection.
+
```

Comment:
This API makes no sense to me. The core should know this and set the state accordingly. The better API would be to enable the plugin to control ticket use for an inbound connection but core should handle the ticket and tracking whether it was used. This is the same reason the cqtr log tag makes sense without any API to set that state.